crates/red_knot_python_semantic/src/types/call/bind.rs

```diff
@@ -568,6 +568,21 @@ impl<'db> Bindings<'db> {
                         }
                     }
 
+                    Some(KnownFunction::GetProtocolMembers) => {
+                        if let [Some(Type::ClassLiteral(class))] = overload.parameter_types() {
+                            if let Some(protocol_class) = class.into_protocol_class(db) {
+                                overload.set_return_type(Type::Tuple(TupleType::new(
```

Comment:
TODO here also for the use of `Type::Tuple`? I think ideally for maximum clarity to future-us, we have TODOs both on tests that are known-wrong, and by the specific part of the implementation that is known wrong.